packages/dashboard-frontend/src/containers/__tests__/FactoryLoader.spec.tsx

```diff
@@ -133,6 +133,188 @@ describe('Factory Loader container', () => {
     jest.useRealTimers();
   });
 
+  describe('converting devfiles', () => {
+    it('should NOT convert devfile v1 in Che Server mode', async () => {
+      const location = 'http://test-location';
+      const workspaceV1 = new CheWorkspaceBuilder()
+        .withId('my-workspace-id')
+        .withName('my-project')
+        .build();
+      const workspace = convertWorkspace(workspaceV1);
+
+      const store = new FakeStoreBuilder()
+        .withCheWorkspaces({
+          workspaces: [workspaceV1],
+        })
+        .withWorkspaces({
+          workspaceId: workspace.id,
+          namespace: workspace.namespace,
+          workspaceName: workspace.name,
+        })
+        .withWorkspacesSettings({
+          'che.devworkspaces.enabled': 'false',
+        } as che.WorkspaceSettings)
+        .withFactoryResolver({
+          v: '4.0',
+          source: 'devfile.yaml',
+          devfile: workspace.devfile,
+          location: location.split('&')[0],
```

Comment:
Do we really need this ```.split('&')[0]``` ?

Reply:
That's copy-pasted part, it's not needed for that unit tests